python/lsst/pipe/tasks/healSparseMapping.py

```diff
@@ -250,6 +251,10 @@ def mask_warp_bbox(self, bbox, visit, mask, bit_mask_value):
         if self.ccd_input_map is None:
             raise RuntimeError("Must run build_ccd_input_map before mask_warp_bbox")
 
+        if len(self._ccd_input_pixels) == 0:
```

Comment:
This is perfectly readable, and you don't need to change this one, but in future dev guide prefers using the fact that empty sequences are false: https://developer.lsst.io/python/style.html#comparisons like 
```if not self._ccd_input_pixels:```

Reply:
You know, I never liked that idiom but decided to try it and it was a 100% disastrous failure.  That idiom *does not work with numpy arrays*.